RealmSwift/LinkingObjects.swift

```diff
@@ -220,6 +220,23 @@ public final class LinkingObjects<T: Object>: LinkingObjectsBase {
     /**
      Returns a `Results` containing all the linking objects, but sorted.
 
+     Objects are sorted based on the values of the given key path. For example, to sort a collection of `Student`s from
+     youngest to oldest based on their `age` property, you might call
+     `students.sorted(byKeyPath: "age", ascending: true)`.
```

Comment:
Perhaps a key path example could be added here to highlight that it's now possible to do?

> Or to sort a collection of `Dog`s alphabetically based on their `owner` property's `name` value, you could call `dogs.sorted(byKeyPath: "owner.name", ascending: true)`.